Allow substituting types (#764)
apple/swift-openapi-generator: Examples/type-overrides-example/README.md

# Overriding types

An example project using [Swift OpenAPI Generator](https://github.com/apple/swift-openapi-generator).

> **Disclaimer:** This example is deliberately simplified and is intended for illustrative purposes only.

## Overview

> This example shows how to use the TypeOverrides feature of the Swift OpenAPI Generator.

Reviewer: ```suggestion
This example shows how to use [type overrides](https://swiftpackageindex.com/apple/swift-openapi-generator/documentation/swift-openapi-generator/configuring-the-generator) with Swift OpenAPI Generator.
```

Reviewer: Please document this feature in https://swiftpackageindex.com/apple/swift-openapi-generator/1.8.0/documentation/swift-openapi-generator/configuring-the-generator, from there please also link the proposal which will land at https://swiftpackageindex.com/apple/swift-openapi-generator/documentation/swift-openapi-generator/soar-0014
